src/core/field.js

```diff
@@ -547,7 +547,7 @@ export default class Field {
       this.validator.validate(`#${this.targetOf}`);
     } : (...args) => {
       // if its a DOM event, resolve the value, otherwise use the first parameter as the value.
-      if (args.length === 0 || (isCallable(Event) && args[0] instanceof Event) || (args[0] && args[0].srcElement)) {
+      if (args.length === 0 || args[0] instanceof Event || (args[0] && args[0].srcElement)) {
```

Comment:
```js
args[0] instanceof Event
```
 will fail since the Event is not callable which why the check is there in the first place, plus:

```js
(args[0] && args[0].srcElement)
```

Is IE specific, so I'm not following why this check doesn't pass since all checks are combined with an `OR` operator. These checks do the following:

1- Check if its not a vue event. OR
2- Check if its an Event instance by the modern browsers. OR
3- Check if its IE event instance.

I'm not sure how your code fixes the issue.